Microsoft.Alm.Authentication/TargetUri.cs

```diff
@@ -42,10 +42,12 @@ public string AbsolutePath
         {
             get { return ResolvedUri.AbsolutePath; }
         }
+
```

Comment:
Accessors are not vertically spaced anywhere else in the project, I think we've adopted that style. We should likely just keep it as is.